### ISSUE TITLE
Change Donor option not showing all donors in donation detail page in back-end.

### ISSUE THREAD

**Issue by emgk** [WordImpress/Give#2378]
## Issue Overview
JS Chosen field `Change Donor:` on Donation detail page is not showing fully.
http://www.screencast.com/t/heV3CGNuJ

## Steps to Reproduce (for bugs)
1. Go to any donation.
2. Try to change donor for that donation.
3. There you will see the js chosen field is not showing completly.

## Todos
- [ ] Tests
- [ ] Documentation

## WordPress Environment
<details>
```
Copy and paste the system Info report from **Donations > Tools > System Info** in WordPress admin here.
```
</details>

**Comment by raftaar1191** [comment]
Nice Catch @emgk

**Comment by ravinderk** [comment]
This is already fixed with #2412
